tests/check/check_test.go

```diff
@@ -41,14 +42,19 @@ type assertion struct {
 
 func TestCheckMemory(t *testing.T) {
 	testCheck(t, "memory")
```

Comment:
Let's compose these in a single `RunAll(t, engine string)` and then call that instead. This way we share all of the tests across invocations.

```
func TestCheckMemory(t *testing.T) {
    RunAll(t, "memory")
}

func TestCheckPostgres(t *testing.T) {
    RunAll(t, "postgres")
}

...
```

Reply:
updated